### DISCUSSION TITLE
Wican Odb on Kia Niro PHEV with Mqtt

### DISCUSSION THREAD

**Discussion by rennai35000** [meatpiHQ/wican-fw#84] [Q&A]
Hello,
I'm new here, I bought the wican odb.
I installed it on my Kia Niro PHEV year 2018. The 2 blue and green LEDs are on
I receive the status online offline in mqtt but nothing else.
I think I understood that you have to activate the ECU to the wican but none of the tests I've done work.
Can anyone help me? 
I'm French and only speak a little English, sorry.
Best regards

**Comment by meatpiHQ** [comment]
@rennai35000 Just a couple of notes to get you started. 

First thing you should set the correct CAN bitrate, usually it's 500k or 250K. Also make sure you have your engine ON when testing.

I suggest that you set the protocol to elm327 first and use an app called carscanner to read some data off the car.

This issue has some good information for you: https://github.com/meatpiHQ/wican-fw/issues/17
